src/sources/redis/mod.rs

```diff
@@ -126,6 +127,10 @@ pub struct RedisSourceConfig {
     #[serde(default = "default_decoding")]
     #[derivative(Default(value = "default_decoding()"))]
     decoding: DeserializerConfig,
+
+    /// The namespace to use for logs. This overrides the global setting
```

Comment:
```suggestion
    /// The namespace to use for logs. This overrides the global setting.
```

Reply:
Haha, yeah I noticed this when I copied the snippet from `datadog_agent` we'll need to change it over there :)